src/DatabaseSchema.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Backpack\CRUD;
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\LazyCollection;
+
+class DatabaseSchema
```

Comment:
Here's a wild idea @pxpm ... What if we do...

```suggestion
final class DatabaseSchema
```

👀🤯 i know... 😅

But this would give us the confidence nobody's going to use this, other than us... so we can refactor this at any time with zero breaking change... 

Am I right? Any reason why we'd let anyone extend this?

Reply:
No reason at all, and I have no worries about it since we can "un-final it" if ever needed. 
Usually final classes rely on an interface implementation, as the name states, they are "the final implementation" of that interface, so if you want a different behaviour you should create your own interface and implement it your way. 
I think this case falls into the "special" cases where an interface is not needed to make it final since it is already a concrete implementation of the concept and can't vary in behaviour. 

I've just hit "Commit suggestion".